### ISSUE TITLE
About scaled flow

### ISSUE THREAD

**Issue by shuosir** [gangsterless/pytorch-softmax-splatting#5]
Hi, there!
Why you put a coefficient(like 20) in front of the scaled flow?

**Comment by gangsterless** [comment]
well ,I follow code from [here](https://github.com/sniklaus/softmax-splatting/issues/12)
